src/Pelagos/Bundle/AppBundle/Resources/views/Account/AccountExists.html.twig

```diff
@@ -3,7 +3,6 @@
 {% block body %}
     <div class="messages error alert alert-danger">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
         The email address you have entered is already associated with a GRIIDC account. If you have forgotten your password or would like to create a new password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}" class="alert-link">password reset</a>. For any additional questions please contact <a href=
             "griidc@gomri.org" class="alert-link">griidc@gomri.org</a>.
```

Comment:
These alert link are bootstrap also, it wouldn't hurt it they stayed, but what's worse is that the link is wrong, it should have email: in front of it. So we can add it, or remove the A tag.